lib/bowermanager.js

```diff
@@ -2,10 +2,7 @@ var _ = require('underscore');
 var async = require('async');
 var bower = require('bower');
 var fs = require('fs-extra');
-var mkdirp = require('mkdirp');
-var ncp = require('ncp').ncp;
 var path = require('path');
-var rimraf = require('rimraf');
 var semver = require('semver');
 
 var installHelpers = require('./installHelpers');
```

Comment:
If we are going to node 8 and dropping 4 could we convert var's like this to `const`. Im not suggesting we do everything but areas as we touch them.